parts/common/components.json

```diff
@@ -538,6 +538,17 @@
       ],
```

Comment:
Keeping the old ip-masq-agent-v2 image for now in case we need to rollback, but we can remove v0.1.14 if we need space

Reply:
Can we remove the older version in this PR? And then the one from the older image source in a subsequent one?

Reply:
Ideally we remove the older image source in this same PR because caching is an optimization. But I understand your concern too.

Reply:
updated to remove old v0.1.14